_includes/glossary/README.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Glossary
+
+To write a glossary snippet, start with a copy of the template in this directory, `_glossary-template.md`. Make sure to provide (or omit) values for the following fields:
+
+* (Required) `term`
+* (Optional) `formerly` and `related`. If you do not provide these values, remove the fields.
+* (Required) short description. Make sure to replace the instructional text in the template with your content.
+* (Optional) long description. If you do not provide a long description, remove the `{% capture %}` and `{% endcapture %}` tags together with the instructional text.
+
+The template also contains basic information about how to write your snippet. For additional guidance, read on in this readme.
+
+## Glossary snippet style guide
+
+This style guide supplements the guidance provided in the glossary template. It's intended to help you think about what and how to write glossary definitions. For more general guidance on style, consult [the core docs style guide](https://kubernetes.io/docs/home/contribute/style-guide/), also available in this repo at `docs/home/contribute/style-guide.md`.
```

Comment:
nit: I don't think it's necessary to mention the path of the style guide (given that you've already linked it), but let me know if you feel otherwise